dashboard/test/ui/runner.rb

```diff
@@ -283,12 +283,19 @@ def output_synopsis(output_text)
     end
   end
 
-  if succeeded
-    HipChat.log "<b>dashboard</b> UI tests passed with <b>#{test_run_string}</b> (#{format_duration(test_duration)})"
+  parsed_output = output_stdout.match(/^(?<scenarios>\d+) scenarios?( \((?<info>.*?)\))?/)
+  scenario_count = parsed_output[:scenarios].to_i
+  scenario_info = parsed_output[:info]
+  scenario_info = ", #{scenario_info}" unless scenario_info.blank?
+
+  if scenario_count == 0
```

Comment:
is it possible to fail with scenario_count == 0? If so we should say failed instead of skipped here.

Also with our new not-spamming-hipchat thing we should decide whether to hipchat log skips or not (probably not because it's expected?).